haddock-api/src/Haddock/Interface.hs

```diff
@@ -120,34 +122,55 @@ processModules verbosity modules flags extIfaces = do
 -- * Module typechecking and Interface creation
 --------------------------------------------------------------------------------
 
-
 createIfaces :: Verbosity -> [String] -> [Flag] -> InstIfaceMap -> Ghc ([Interface], ModuleSet)
 createIfaces verbosity modules flags instIfaceMap = do
   -- Ask GHC to tell us what the module graph is
   targets <- mapM (\filePath -> guessTarget filePath Nothing) modules
   setTargets targets
-  modGraph <- depanal [] False
+  _modGraph <- depanal [] False
```

Comment:
I'm not sure why without this line the test won't pass.

I'm also not sure if this will break the parallelization.

Reply:
`depanal` will change the state hold by `Ghc` Monad.